Formula/r/rip2.rb

```diff
@@ -0,0 +1,28 @@
+class Rip < Formula
```

Comment:
```suggestion
class Rip2 < Formula
```
Since the formula file is called `rip2.rb`.